furusiya/function_mess.py

```diff
@@ -122,7 +122,7 @@ def process_bow():
                 elif event.type == 'KEYDOWN':
                     if event.key == 'ESCAPE':
                         Game.draw_bowsight = False
-                        break
+                        return 'didnt-take-turn'
```

Comment:
This is now a constant. We shouldn't be using strings.

Please double-check that this doesn't break if you:
- Press `f`
- Press escape

Reply:
Nope. Works as expected.

However, that is a good point. I'll open an issue to convert constant-ish strings into enums.